app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        Person(name=person["name"], age=person["age"])
+
+    for person in people:
+
+        if "husband" in person and person.get("husband") is not None:
+            Person.people[
+                person["name"]].husband = Person.people[person["husband"]]
```

Comment:
Braces should be located in one of two options: open and start with the text or have a line break between the text. You have both options now

Reply:
Try to split on variables, it would be even better